### PR TITLE
adding web bounty hall of fame page

### PR DESCRIPTION
**Pull request by rforbes**

### CODE REVIEW

File: vendor-local/src/legal-docs [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit fdf03d97dff7a53e6331018649c357fa6eea9b9e
+Subproject commit 21758916986f033b4368f6311b0acbc2bdbeb0dd
```
Comment (pmclanahan) [inline vendor-local/src/legal-docs]: Looks like this sneaked into your commit.

**Comment by pmclanahan** [comment]
LGTM apart from fixing the legal-docs submodule change.

**Comment by schalkneethling** [comment]
@rforbes see if this helps - http://schalkneethling.github.io/blog/2015/08/24/there-is-submodule-commit-in-your-pr/ getting rid of that submodule

**Comment by craigcook** [comment]
r+ on the front-end bits. :maple_leaf:

**Comment by jgmize** [comment]
After @craigcook's r+ I ammended the commit to eliminate the submodule change and merged in cfad2ceaeedb189215f301fe7d8c98d614ecc19a

**Comment by jgmize** [comment]
Steps taken, for future reference:
```
git co master
git pull
git pr 3362
git co pr/3362
git co master -- vendor-local/src/legal-docs
git ci --amend
git rebase master
git co master
git merge pr/3362
git push origin master
```
Several of the commands above use aliases defined in my [.gitconfig](https://github.com/jgmize/dotfiles/blob/master/.gitconfig)

**Pull request closed without merging**